test/fail_compilation/fail18417.d

```diff
@@ -1,4 +1,4 @@
-// REQUIRED_ARGS : -de
+// REQUIRED_ARGS: -de
```

Comment:
If the space before : is allowed or not is not documented anywhere.

Reply:
Fixed